### PR TITLE
feat(client-debugger-chrome-extension): Restyled the Telemetry view and added expandable info column

### PR DESCRIPTION
**Pull request by WayneFerrao**
[AB#3870](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/3870)

## Description
This PR restyles the Telemetry view to a data grid with 3 primary columns. 
image

### CODE REVIEW

File: packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx [hunk 1]
```diff
@@ -104,10 +117,121 @@ export function TelemetryView(): React.ReactElement {
...
+	const items: Item[] =
+		telemetryEvents !== undefined
+			? telemetryEvents?.map((message) => {
+					return {
+						category: message.logContent.category,
+						eventName: message.logContent.eventName,
+						information: JSON.stringify(
+							message.logContent,
+							jsonSerializationTransformer,
+							"  ",
```
Comment (jikim-msft) [inline packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx:138]: What is this for?

File: packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx [hunk 2]
```diff
@@ -104,10 +117,121 @@ export function TelemetryView(): React.ReactElement {
...
+				);
+			},
+		}),
+		createTableColumn<Item>({
+			columnId: "information",
+			renderHeaderCell: () => {
+				return <b>Information</b>;
+			},
+			renderCell: (message) => {
+				// eslint-disable-next-line react-hooks/rules-of-hooks
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx:173]: I think you need to hoist this up to the component root (per the linter rule), or you'll want to extract the cell rendering logic into its own component. As is, I believe this violates React state requirements.

Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx:173]: Actually, I might be wrong there. I guess it depends on how the table component is calling `renderCell`. If it's being rendered like a React component, then this is effectively just an inline component, in which case there isn't a problem. Even still, probably best to extract the logic into its own component to be safe (and so we don't have to disable our linter rules).

File: packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx [hunk 3]
```diff
@@ -104,10 +117,121 @@ export function TelemetryView(): React.ReactElement {
...
+		}),
+		createTableColumn<Item>({
+			columnId: "eventName",
+			renderHeaderCell: () => {
+				return <b>Event Name</b>;
+			},
+			renderCell: (message) => {
+				return (
+					<div style={{ maxWidth: 160, whiteSpace: "normal", wordWrap: "break-word" }}>
+						{message.eventName.slice("fluid:telemetry:".length)}
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx:162]: Can we add a brief comment explaining what we're doing here? Will make it easier to find this if we choose to change it in the future.

File: packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx (hunk 1 above)
Comment (WayneFerrao) [reply packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx:138]: @jikim-msft  It was a json serializer that removed values that were already being displayed ( logcontent.category/ eventName) before showing the json. But this is being deprecated and all json values in information will be shown.

**Review by alexvy86 — APPROVED** [review]

**Pull request merged**